app/models/hold.rb

```diff
@@ -19,16 +19,16 @@ def check_conflicting_rentals
   end
 
   def replace_rental(curr_rental)
-    new_rental = Rental.create(item_type_id: curr_rental.item_type_id,
-                               user_id: curr_rental.user_id, department_id: curr_rental.department_id,
-                               start_time: curr_rental.start_time, end_time: curr_rental.end_time)
-
-    curr_rental.cancel
-    curr_rental.save
-    ReplacementMailer.replacement_email(curr_rental.user, self, curr_rental, new_rental).deliver_now
+    new_rental = Rental.new(item_type_id: curr_rental.item_type_id, creator_id: curr_rental.creator_id,
+                            renter_id: curr_rental.renter_id, department_id: curr_rental.department_id,
+                            start_time: curr_rental.start_time, end_time: curr_rental.end_time)
+    new_rental.create_reservation
+    new_rental.save
```

Comment:
what if this fails? on `new_rental.save` failure it will simply return `false` right?